src/ORM/DataObject.php

```diff
@@ -3495,7 +3495,7 @@ public function searchableFields()
      *
      * @param boolean $includerelations a boolean value to indicate if the labels returned include relation fields
      *
-     * @return array|string Array of all element labels if no argument given, otherwise the label of the field
+     * @return array Array of all element labels if no argument given, otherwise the label of the field
```

Comment:
The "otherwise" part is the bit that was where it said it'd return a string right? Should we remove that too?

Reply:
Done